ts/components/PreferenceItem.tsx

```diff
@@ -4,8 +4,8 @@ import * as React from "react";
 import { connectStyle } from "native-base-shoutem-theme";
 import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
 
-import IconFont from "../components/ui/IconFont";
 import variables from "../theme/variables";
+import IconFont from "./ui/IconFont";
```

Comment:
Somebody didn't run `prettier`. Should I revert this changes?

Reply:
what do you mean?

Reply:
Running `prettier` it changed several files I didn't edit. Should I revert the changes `prettier` applied or I leave them?

Reply:
either you have a different prettier configuration or the tslint check in the circleci build failed to detect the prettier issues... anyway, you should not commit changes not related to the story you're fixing